### PR TITLE
Input: support resize prop

### PR DESCRIPTION
**Pull request by Kingwl**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: examples/docs/zh-CN/input.md [hunk 1]
```diff
@@ -728,6 +728,7 @@ export default {
 | name | 原生属性 | string | — | — |
 | max | 原生属性，设置最大值 | * | — | — |
 | min | 原生属性，设置最小值 | * | — | — |
+| resize | 控制是否能被用户缩放 | string | none, both, horizontal, vertical | — |
```
Comment (QingWei-Li) [inline examples/docs/zh-CN/input.md]: 英文文档也麻烦补充了吧, 比心

Comment (Kingwl) [inline examples/docs/zh-CN/input.md]: done

Comment (reverland) [inline examples/docs/zh-CN/input.md]: @Kingwl 我们有这个需求？

Comment (Kingwl) [inline examples/docs/zh-CN/input.md]: 其实是有的....

**Pull request merged**